examples/read_a_single_parameter/main.tf

```diff
@@ -1,16 +1,36 @@
-# Configure AWS
+# Default AWS provider (required to set region, but not used)
```

Comment:
I didn't realize that you would have to define the default provider even if you don't use it.  This makes me think that we should make one of the other providers the default.

Reply:
Thanks for mentioning this.  I had thought about doing exactly that, but needed another voice to push me over the edge.  This has now been changed in https://github.com/cisagov/molecule-packer-ci-iam-user-tf-module/pull/8/commits/9b3eb1d2f076d02e2dcf168cb7788ab0f902c028.